### PR TITLE
Fix material design icon sizes

### PR DESCRIPTION
**Pull request by marcoambrosini**
There were still some 24px icons around, this pr gets rid of those.

Signed-off-by: marco <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 1]
```diff
@@ -195,11 +195,13 @@
...
 				@click.stop="toggleHandRaised">
+				<!-- The following icon is much bigger than all the others
+						so we reduce its size -->
 				<HandBackLeft
-					:size="20"
+					decorative
 					title=""
-					fill-color="#ffffff"
-					decorative />
+					size="18px"
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:203]: 💥 
Bildschirmfoto von 2022-01-12 14-04-45

```suggestion
					size="18"
```

You didn't try this?

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 2]
```diff
@@ -209,11 +211,13 @@
...
 					@click="toggleHandRaised">
+					<!-- The following icon is much bigger than all the others
+						so we reduce its size -->
 					<HandBackLeft
-						slot="icon"
-						:size="20"
 						decorative
-						title="" />
+						title=""
+						size="18px"
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:219]:
```suggestion
						size="18"
```

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 3]
```diff
@@ -84,24 +84,27 @@
...
 				decorative />
+			<!-- The following icon is much bigger than all the others
+						so we reduce its size -->
 			<HandBackLeft
 				v-if="callIcon === 'hand'"
-				:size="24"
+				decorative
 				title=""
-				decorative />
+				size="18px"
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue:106]:
```suggestion
				size="18"
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by marcoambrosini** [comment]
😅

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 4]
```diff
@@ -209,11 +211,13 @@
 				<ActionButton
 					:close-after-click="true"
 					@click="toggleHandRaised">
+					<!-- The following icon is much bigger than all the others
+						so we reduce its size -->
 					<HandBackLeft
-						slot="icon"
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:213]: This breaks it still:

> [Vue warn]: Error in data(): "TypeError: this.$slots.default[0].text is undefined"

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**